.github/workflows/unit_tests.yml

```diff
@@ -57,6 +57,7 @@ jobs:
         pip install kaldi_native_io  # for running kaldi_native_io tests
         pip install webdataset==0.2.5 # for running webdataset tests
         pip install git+https://github.com/fgnt/nara_wpe  # for running WPE tests
+        pip install num2words
```

Comment:
I'd rather not add it to the CI, since we are not testing torchaudio forced alignment here. I don't want to make `num2words` a dependency of Lhotse, i.e. the code has to be able to import correctly without `num2words` being installed. I'd like the CI to test that.